src/WebJobs.Script/Rpc/Configuration/WorkerConfigFactory.cs

```diff
@@ -131,14 +129,6 @@ internal void AddProvider(string workerDir)
                 var languageSection = _config.GetSection($"{LanguageWorkerConstants.LanguageWorkersSectionName}:{workerDescription.Language}");
                 workerDescription.Arguments = workerDescription.Arguments ?? new List<string>();
 
-                descriptionProfiles = GetWorkerDescriptionProfiles(workerConfig);
-                if (ScriptSettingsManager.Instance.IsAppServiceEnvironment)
-                {
-                    // Overwrite default Description with AppServiceEnv profile
-                    // TODO:pgopa delete after ANT78
-                    workerDescription = GetWorkerDescriptionFromProfiles(LanguageWorkerConstants.WorkerDescriptionAppServiceEnvProfileName, descriptionProfiles, workerDescription);
```

Comment:
Can we also remove `WorkerDescriptionAppServiceEnvProfileName` from LangageWorkerConstants?

Reply:
thanks! fixed.

Reply:
And updated the description. ANT83 has fix that removes dependency on FunctionsJava